src/util.ts

```diff
@@ -47,3 +51,32 @@ export function getFileExtension(pathOrUrl: string): string {
         pathOrUrl
     );
 }
+
+export function isBinarySimulariumFile(fileContents: Blob): Promise<boolean> {
+    const first16blob: Blob = fileContents.slice(0, 16);
+    // compare this with "SIMULARIUMBINARY"
+    return first16blob.text().then((text) => {
+        return text === "SIMULARIUMBINARY";
```

Comment:
You make this a constant as "secret", is there a reason you're using the string here instead of exporting that constant?

Reply:
Yes, I'll do something better to remove this redundancy.